### PR TITLE
Loops

### PR DESCRIPTION
**Pull request by antonredko1993**

### CODE REVIEW

File: ElementaryTasks/Loops.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+{
+    class Loops
+    {
+        public void SumOfEvenValues()
+        {
+            int sum = 0;
+            int count = 0;
+            for (int i = 1; i < 99; i++)
+            {
+                if (i % 2 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Please revert statement to remove 'continue'

File: ElementaryTasks/Loops.cs [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            Console.WriteLine($"Summ: {sum}");
+            Console.WriteLine($"Count: {count}");
+        }
+
+        public void PrimeNumber()
+        {
+            Console.WriteLine("Enter the value");
+            double.TryParse(Console.ReadLine(), out double userInput1);
+
+            if (userInput1 <= 1 ^ userInput1 % 1 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Please check condition

File: ElementaryTasks/Loops.cs [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                Console.WriteLine($"Value: {i}");
+            }
+            Console.WriteLine($"Summ: {sum}");
+            Console.WriteLine($"Count: {count}");
+        }
+
+        public void PrimeNumber()
+        {
+            Console.WriteLine("Enter the value");
+            double.TryParse(Console.ReadLine(), out double userInput1);
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Why double?

File: ElementaryTasks/Loops.cs [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+                Console.WriteLine($"{ userInput1} is a prime number");
+            }
+        }
+
+        public void Square()
+        {
+            Console.WriteLine("Enter the value");
+            int.TryParse(Console.ReadLine(), out int userInput);
+
+            if (userInput <= 1 ^ userInput % 1 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Check condition

File: ElementaryTasks/Loops.cs [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+                }
+            }
+        }
+
+        public void Chess()
+        {
+            Console.WriteLine("Enter number of rows");
+            int.TryParse(Console.ReadLine(), out int userInput1);
+
+            if (userInput1 <= 1 ^ userInput1 % 1 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Check condition

File: ElementaryTasks/Loops.cs [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+                Console.WriteLine($"{userInput1} isn't correct value");
+            }
+            else
+            {
+                Console.WriteLine("Enter number of columns");
+            }
+
+            int.TryParse(Console.ReadLine(), out int userInput2);
+
+            if (userInput2 <= 1 ^ userInput2 % 1 != 0)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Check condition

File: ElementaryTasks/Loops.cs (hunk 1 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 3 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 2 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 4 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 5 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 6 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

**Pull request merged**